blues/application/tasks.py

```diff
@@ -224,24 +224,54 @@ def generate_nginx_conf(role='www'):
         f.write(conf)
 
 
-def notify_deploy(role=None):
+def get_github_org():
+    from .project import git_repository
+
+    url = git_repository()
+    if 'url' not in url or not url.startswith('git@github.com:'):
+        return None
+
+    domain, repo = url.split(':')
+    org, repo = repo.split('/')
+    return org
+
+
+def notify_deploy(role=None, commits=None):
     from .project import project_name, git_repository_path
 
+    msg = '`{deployer}` deployed '
+    if role:
+        msg += '`{project}::{state}:{role}` '
```

Comment:
Also, with

```python
'role': role or '',
```
there will always be a trailing colon when there isn't any role, which I think isn't particularly nice.